specification/implementation/metadata/mets/amdsec/index.md

```diff
@@ -1,10 +1,12 @@
 ### 5.3.4.	Use of the METS administrative metadata section (element amdSec)
 The purpose of the METS administrative data section is to embed or refer to files containing administrative metadata about the IP content objects. CSIP is only using referencing of files containing administrative metadata. The CSIP (and METS) categorises preservation metadata as administrative metadata, specifically Digital Provenance metadata (following the avaiable guidelines), hence all preservation metadata should be referenced from a digiprovMD element within the amdSec.
 
-The METS amdSec element must include references to all relevant metadata located in the folder “metadata/preservation”. This means also that the root level `METS.xml`
+The administrative metadata section also contains information regarding rights statements. Regardless of which standard that is used for registering the rights metadata all the rights metadata should be reference from a rightsMD element within the amdSec. Standards that can be used includes the collaborative work from RIGHTSTATMENTS.ORG <http://rightsstatements.org> , Europena right statements <https://pro.europeana.eu/page/available-rights-statements> , the METS Rights Schema <https://github.com/mets/METS-Rights-Schema> created and maintaned by the METS Board, the rights part of PREMIS <http://www.loc.gov/standards/premis/> as well as own local rights statements in use.</p>
```

Comment:
The administrative metadata section also contains rights statements. Regardless of used standard used to record the rights metadata it should be referenced from a rightsMD element within the amdSec. Suitable standards include the collaborative work from RIGHTSTATMENTS.ORG <http://rightsstatements.org> , Europena right statements <https://pro.europeana.eu/page/available-rights-statements> , the METS Rights Schema <https://github.com/mets/METS-Rights-Schema> created and maintained by the METS Board, the rights sections of PREMIS <http://www.loc.gov/standards/premis/> as well as any local rights statements.